Azure Services/Azure Monitor/Agents/Migration Tools/DCR Config Generator/WorkspaceConfigToDCRMigrationTool.ps1

```diff
@@ -197,15 +236,18 @@ function Get-DCRBaseJson
 
     $dataSources = Get-DataSources -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -PlatformType $PlatformType
     $destinations = Get-Destinations -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName
-    $dataFlows = [System.Collections.ArrayList]@(Get-DataFlows -WorkspaceName $WorkspaceName -PlatformType $PlatformType)
+    $dataFlows = [System.Collections.ArrayList]@(Get-DataFlows -WorkspaceName $WorkspaceName -PlatformType $PlatformType -ResourceGroupName $ResourceGroupName)
+    $streamDeclarations = Get-CustomLogStreamDeclarations -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName
 
     if(-not (Get-DataSourceIsEmpty -DataSource $dataSources))
     {
         $properties = 
         [ordered]@{
+            "dataCollectionEndpointId" = "[parameters('dataCollectionEndpoint_id')]";
+            "streamDeclarations" = $streamDeclarations;
```

Comment:
If custom logs is not set up in MMA, streamDeclarations would be null. Is that okay?

Reply:
Same comment as above

Reply:
Verified - Same comment as above